ConsoleApp/Stack.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using static System.Console;
+namespace ConsoleApp;
+
+class Stack<T>
+{
+    private record ListItem
+    {
+        public T Value { get; init; }
+        public ListItem Previous { get; set; }
+    }
+
+    private ListItem _last;
+    public int Height { get; private set; }
```

Comment:
it should be Count instead
```suggestion
    public int Count { get; private set; }
```

Reply:
hm.. but it's the method name also.. 
![image](https://user-images.githubusercontent.com/106695465/179757892-0f495c44-31dd-42d5-a159-cc9b5be519bf.png)